spec/support/page_objects/trainees/confirm_details.rb

```diff
@@ -4,5 +4,9 @@ class ConfirmDetails < PageObjects::Base
       set_url "/trainees/{id}/{section}/confirm"
       element :submit_button, "input[name='commit']"
     end
+
+    class ConfirmDiversityDetails < ConfirmDetails
```

Comment:
shouldn't it be on its own?

Reply:
yeah i agree, done https://github.com/DFE-Digital/register-trainee-teachers/pull/142/commits/f09be03ac01c0f033bdc4cf56fb709b64bbe222f